css/styles.css

```diff
@@ -0,0 +1,299 @@
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.banner-image1 {
+  width: 222.69px;
+  height: 192px;
+}
+
+.banner-image2 {
+  width: 222.62px;
+  height: 270.73px;
+}
+
+.banner-image3 {
+  width: 222.62px;
+  height: 231.39px;
+}
+
+.banner-image4 {
+  width: 222.95px;
+  height: 231.51px;
+}
+
+.banner-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 60px;
+}
+
+.banner-header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 20px;
+
+  width: 588px;
+  height: 195px;
+}
+
+.logo-companies {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.logo {
+  width: 17px;
+  height: 25.02px;
+}
+
+.banner-header h2 {
+  font-weight: 600;
+  font-size: 60px;
+  line-height: 108%;
+  letter-spacing: -1px;
+  background: var(--primary-gradient);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.banner-header h2 span {
+  -webkit-text-fill-color: var(--black);
+}
+
+.banner-buttons {
+  display: flex;
+  gap: 21px;
+  margin-top: 60px;
+}
+
+.register-here {
+  width: 225px;
+  height: 36px;
+  border-radius: 5px;
+  border: 0px;
+  background: var(--primary-gradient);
+
+  color: var(--white);
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.25);
+
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.tutorial {
+  border: 1px solid var(--black);
+  border-radius: 5px;
+  background-color: var(--white);
+  box-shadow: none;
+
+  width: 223px;
+  height: 36px;
+
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 108%;
+  letter-spacing: -1px;
+
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.banner-for-people {
+  display: flex;
+  flex-direction: column;
+  margin-top: 15px;
+}
+
+.banner-for-people h4 {
+  font-weight: 300;
+  line-height: 130%;
+
+  letter-spacing: -1px;
+}
+
+.people-title {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.people-title h4 {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 108%;
+}
+
+.banner-for-people p {
+  width: 345px;
+  height: 42px;
+
+  font-weight: 300;
+  line-height: 130%;
+  letter-spacing: -1px;
+}
+
+.partner-companies {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 229px;
+  gap: 60px;
+}
+
+.partner-companies-logos {
+  display: flex;
+  flex-wrap: wrap;
+  width: 900px;
+  gap: 16px;
+}
+
+.partner-companies-images {
+  width: 94px;
+  height: 25px;
+  margin-bottom: 10px;
+}
+
+.partner-companies h5 {
+  text-align: center;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 108%;
+  letter-spacing: -1px;
+
+  width: 709px;
+  height: 68px;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 229px;
+  gap: 200px;
+}
+
+.highlights1 {
+  display: flex;
+  align-items: center;
+  gap: 141px;
+}
+
+.highlights-AT {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+
+  width: 584px;
+  height: 234px;
```

Comment:
Don't think we need the height here. Regarding the width, not really sure if we need to specify it here, but if that's the case, I would do it in % instead of px as we discussed.